starter_code/index.html

```diff
@@ -5,21 +5,74 @@
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
+		<title>Relaxr</title>
 
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
 
 	</head>
+	
 	<body>
-		<div id="container">
+		
+		<div class="Id">
 			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+			<h1>Relaxr</h1>
+				<h2>Get piece of mind with a single tap</h2>
+		
+				<div class="btnClass">	
+					<button>Get it Now</button>
+				</div>		
 			</header>
+
+			<section class="first">
+						<h1>Benefits</h1>
+							<p>The perfect personal assistant. Relaxr does your job<br>
+							for you so you can enjoy life as it is meant to be.</P>
+							
+							<ul>
+								<li>Schedule meeting for you</li>
+								<li>Exel automation to complete your work for you</li>
+								<li>Respond to emails on your behalf</li>
+								<li>Does all your work for you with our revolutionary AI</li>
+							</ul>
+						
+			</section>
+					
+			<div class="quote-container">
+					<header>
+						<p style="text-align: center" font-style="italic">"Relaxr changed my life. I've been able <br> to travel the world, spend limited time<br>
+						working and my boss keeps thanking<br> me for crushing work"<br><strong>-Amanda, Intuit</strong></p>
+					</header>
+				</div>		
+							
```

Comment:
@joeyluca2003 there's no need of this much space to separate the tags. You can use a few lines but always keep the indentation and don't leave a visible gap.